cilium/ebpf.io-summit
# fix: update FB logo on Members page

**Describe what changes this pull request brings**
This pull request brings the FB logo update on Members page




**Steps to test**
1. Open [the page](https://deploy-preview-14--ebpf-summit-2021.netlify.app/members/)
2. Confirm that everything looks and works as expected


**Screenshots**
![image](https://user-images.githubusercontent.com/48465000/129205696-7e117ee1-649e-4666-881e-b1de3e7ca41f.png)



**Checklist**
